controller/controller.go

```diff
@@ -0,0 +1,147 @@
+package controller
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"github.com/dgrijalva/jwt-go"
+	"iotdashboard/dbmanager"
+	"golang.org/x/crypto/bcrypt"
+	"time"
+)
+
+//TODO: these probably shouldn't be global vars
```

Comment:
I agree. A typical pattern in Go is you make a "service" struct/object and have all parameters required for its operation as its fields. I.e. you can have a 

```go
type DashboardController struct {
  psql   dbmanager.DBManager
  jwtKey []byte
  ...
}
```

as an example.